scipy/sparse/csgraph/_flow.pyx

```diff
@@ -407,3 +426,207 @@ cdef ITYPE_t[:] _edmonds_karp(
             # If no augmenting path could be found, we're done.
             break
     return flow
+
+@cython.boundscheck(False)
+@cython.wraparound(False)
```

Comment:
If relevant, you can [set those directives globally](https://cython.readthedocs.io/en/latest/src/userguide/source_files_and_compilation.html#globally) at the top of the file.